source/blood/src/common.cpp

```diff
@@ -308,26 +308,15 @@ static int G_ReadRegistryValue(char const * const SubKey, char const * const Val
 static void G_AddSteamPaths(const char *basepath)
 {
     UNREFERENCED_PARAMETER(basepath);
-#if 0
     char buf[BMAX_PATH];
 
-    // PORT-TODO:
-    // Duke Nukem 3D: Megaton Edition (Steam)
-    Bsnprintf(buf, sizeof(buf), "%s/steamapps/common/Duke Nukem 3D/gameroot", basepath);
-    addsearchpath(buf);
-    Bsnprintf(buf, sizeof(buf), "%s/steamapps/common/Duke Nukem 3D/gameroot/addons/dc", basepath);
-    addsearchpath_user(buf, SEARCHPATH_REMOVE);
-    Bsnprintf(buf, sizeof(buf), "%s/steamapps/common/Duke Nukem 3D/gameroot/addons/nw", basepath);
-    addsearchpath_user(buf, SEARCHPATH_REMOVE);
-    Bsnprintf(buf, sizeof(buf), "%s/steamapps/common/Duke Nukem 3D/gameroot/addons/vacation", basepath);
+    // Blood: Fresh Supply
+    Bsnprintf(buf, sizeof(buf), "%s/steamapps/common/Blood", basepath);
     addsearchpath_user(buf, SEARCHPATH_REMOVE);
```

Comment:
I believe this line should be changed to `addsearchpath(buf);`. Semantically, `SEARCHPATH_REMOVE` means the folder should no longer be considered for file reading after the RFF is loaded, which is never valid for Blood due to various files being loose in the directory, as well as CD music. This flag is used for Duke and SW to keep the virtual search path as clean as possible. If you don't observe any issue, it is probably because NBlood does not call the function to remove paths with this flag, but my point stands.

Reply:
I was unaware of the difference of the two and just went with one of them
I will change this ASAP